README.md

```diff
@@ -51,6 +51,17 @@ Cluster configuration
 Find your cluster below, and add the corresponding suggested setup instructions
 to `~/.bash_profile`, `~/.bashrc` or similar shell startup file.
 
+
+### Perlmutter @ NERSC
+
+```
+module load python
+module load cudatoolkit
+module load craype-accel-nvidia80
+module load cray-pmi
+module del cray-libsci
+```
+
```

Comment:
Can you also make a note that on Perlmutter users should use a prefix rather than a name for conda environments, to make sure envs are installed under `/global/common`, which is visible on all the nodes? E.g. use:

```
conda env create -p $PREFIX -f env.yaml
conda activate $PREFIX
```

Reply:
https://github.com/nv-legate/quickstart/pull/7/commits/61c8fd3734c88a4e2705b5ee359d9a71779db1fe